sdk/identity/identity/managed-identity-matrix.json

```diff
@@ -0,0 +1,20 @@
+{
+    "displayNames": {
+    "**/test-results.xml": ""
+    },
+    "include": [
+      {
+        "Agent": {
+          "msi_image": {
```

Comment:
Even in this file we use both MSI and MI interchangeable. Should we standardize or just use the longer name? `managed_identity_image` for example